python2/python/plan.sh

```diff
@@ -43,6 +40,13 @@ do_prepare() {
   sed -i -re "/(SSL=|_ssl|-DUSE_SSL|-lssl).*/ s|^#||" Modules/Setup.dist
 }
 
+do_setup_environment() {
+   set_buildtime_env PYTHON_SITE_PACKAGES "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
+   set_runtime_env PYTHON_SITE_PACKAGES "${pkg_prefix}/lib/python${pkg_version%.*}/site-packages"
```

Comment:
This is intended to not include `${pkg_prefix}`, as it is for downstream packages to use.
```bash
  set_buildtime_env PYTHON_SITE_PACKAGES "lib/python${pkg_version%.*}/site-packages"
  set_runtime_env PYTHON_SITE_PACKAGES "lib/python${pkg_version%.*}/site-packages"
````